src/homekit.cs

```diff
@@ -1155,7 +1155,7 @@ interface HMAccessoryCategory {
 	[iOS (9,0)]
 	[BaseType (typeof (HMEvent))]
 	[DisableDefaultCtor]
-	interface HMCharacteristicEvent { // NSMutableCopying conformance in header but invalud copyWithZone: and mutableCopyWithZone: selectors on iOS Unified 32-bits
+	interface HMCharacteristicEvent : NSMutableCopying {
```

Comment:
Mmh so we can't specify `HMCharacteristicEvent` implements the protocol on 64 bit only right?

> invalid copyWithZone: and mutableCopyWithZone: selectors on iOS Unified 32-bits

This is gonna be true with your change now. Isn't that a problem?

Reply:
No, protocol conformance varies with iOS versions (that's not new) but intro must know about it (since it can execute on earlier versions where the conformance did not exist).